templates/etc/nsd/nsd.includes.conf.erb

```diff
@@ -0,0 +1,5 @@
+<%- @includes.each do |include| -%>
+  <%- if include != :undef -%>
```

Comment:
Why do you need this if statement?  because of the type enforcement this can only ever be a string

Reply:
Thank you for your feedback.
fixed with https://github.com/icann-dns/puppet-nsd/pull/10/commits/624bcbb9944957a64bef2c8d658a9116ed6be8a4